_posts/platform/deployment/buildpacks/2000-01-01-nginx.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Nginx Buildpack for Custom Reverse Proxy
+nav: Nginx Buildpack
+modified_at: 2018-06-01 00:00:00
+tags: buildpacks build nginx
+---
+
+## Purpose of this buildpack
+
+Your applications are hosted behind a reverse proxy. You might need to customize the way the reverse
```

Comment:
@EtienneM what do you really think is the purpose of this buildpack? :)

Reply:
Customers use it to route different paths to different applications.